### DISCUSSION TITLE
Self-updating internal links

### DISCUSSION THREAD

**Discussion by thumDer** [outline/outline#5219] [Ideas]
Do you have any plans about how to handle page title modifications if they are linked in other pages? Currently they have static titles, generated from their actual title, but updating them won't change their linked form. I understand that there are different viewpoints regarding this, like some people prefer having them in their original form, but others like if they get updated when it is needed.

My idea about serving both user groups:
- if the links text is manually changed from the original page title it should keep that manual title, as long as it is not linked again
- if the links text is still the title of the page it should follow all title updates.

I hope it makes sense, if not please let me know, to ellaborate further.

**Comment by tommoor** [comment]
Previously documents automatically updated the title in backlinked locations however this functionality was lost with the move to collaborative editing, I'd still plan to bring this automatic title updating back eventually.
